meet-data/spf-archive/0447/lifters.csv

```diff
@@ -0,0 +1,33 @@
+Name,Division,BodyweightKg,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Equipment,Event,Sex,WeightClassKg,Place
+Michael Walker,Pre-teen,44,,13.61,36.29,49.9,Raw,BD,M,44,1
```

Comment:
Michael's 85lb bench attempt was dropped. You can create a column "Bench4LBS" and then use the normal csv-tokg script.